src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -992,24 +992,16 @@ public function testWrongSignal()
             $this->markTestSkipped('POSIX signals do not work on Windows');
         }
 
-        if (\PHP_VERSION_ID < 80000 || \is_int($signal)) {
-            $this->expectException(RuntimeException::class);
-        } else {
-            $this->expectException('TypeError');
```

Comment:
Strange, I added this line today.

Reply:
Yes, it made sense on 3.4 where this test is parameterized (string + int). On 4.4 however, only the execution path for the int remained.